operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -400,20 +419,31 @@ Please refer [iSCSI Metrics](https://github.com/Cray-HPE/sbps-marshal/blob/main/
 
 ## Steps to switch from DVS to iSCSI
 
+This involves this sequence of operations:
+
 * Disable DVS
 * Uninstall CPS
 * Enable iSCSI SBPS
 
-To disable DVS please refer:
-<https://github.hpe.com/hpe/hpc-shasta-os-uss-docs/blob/master/dac-assets/documents/operations/DVS_and_SBPS.md>
+To perform these operations, please refer the section `DVS and SBPS` documented in the publication:
+`HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray Supercomputing EX Systems (S-8063)`.
+
+## Steps to continue using DVS based projection
 
-To uninstall CPS services please refer:
-<https://github.hpe.com/hpe/hpc-shasta-os-uss-docs/blob/master/dac-assets/documents/operations/Uninstall_CPS.md>
+If a user wants to continue using DVS, then during the BOS session template creation (manually using `cray bos` command)
+these parameter values have to be used:
 
-To enable iSCSI SBPS please refer:
-<https://github.hpe.com/hpe/hpc-shasta-os-uss-docs/blob/master/dac-assets/documents/operations/Scalable_Boot_projection_service.md>
+```text
+rootfs_provider: "cpss3"
+rootfs_provider_passthrough: "dvs:api-gw-service-nmn.local:300:hsn0,nmn0:0"
+```
+
+If the `sat` command is used to create the BOS session template, then please comment out the two lines marked SBPS and uncomment the above
+two lines marked CPS in `vcs/bootprep/compute-and-uan-bootprep.yaml` and then initiate the compute/UAN node boot. Please refer the section
+`Content Projection Service` in the publication: `HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray
+EX Systems (S-8063)` for more details on DVS based boot content projection of `rootfs`/`PE` images.
 
-Note: These URL references will be replaced with appropriate document name and number in support center.
+**Note:**: Rest of the steps in [Steps to achieve SBPS](#steps-to-achieve-sbps) viz., Image Tagging, Monitor iSCSI metrics need to be skipped.
```

Comment:
What does "rest of the steps" mean?  Are any of the steps relevant if they are using DVS?  If not, the I'd change this to something like 

```suggestion
**Note:**: The steps in [Steps to achieve SBPS](#steps-to-achieve-sbps) are not relevant and should not be followed if DVS is used.
```